explanation/definitions.md

```diff
@@ -0,0 +1,70 @@
+(definitions)=
+# Definitions 
+````{margin}
+```{note}
+This is referency, but these definitions are specifically to help explain in the explanation section, should stay here?
+
+```
+````
+Understanding this explanation section depends on knowing the defintions of several concepts. More can be learned about these concepts from the resources linked in the reference section (link)
+
+
+## Graph Data Concepts
+````{margin}
+```{note}
+Borrowed from brick. I remember finding this very helpful.
+```
+````
+ - **Entity:** An entity is an abstraction of any physical, logical or virtual item; the actual "things" in a building. Examples are mechanical equipment such as air handling units, luminaires, spatial elements like rooms, or the area of these rooms served by certain mechanical equipment. Virtual and logical entities may represent the various data points in a BAS system, such as sensing and actuation points, as well as basic representations of controls.
```

Comment:
Virtual and logical entities are not represented in 223, so I would remove this part

Reply:
Just remove virtual and logical parts, right?
I definitely agree with this change, as using entity to refer to "things that can have properties" in the standard is much more useful than including properties and functionblocks in the definition of entities ( I think those are our representation of virtual and logical elements).

Reply:
Yeah, that should do it